data/coalition/coalition missions.txt

```diff
@@ -143,3 +143,469 @@ mission "Coalition License Patch"
+
+
+mission "Coalition Games 1"
+	name "Coalition Athletes"
+	description "Transport three athletes to <destination> where they will compete in the Coalition games."
+	deadline
+	passengers 3
+	to offer
+		"coalition jobs" >= 35
+		month == 7
+	source
+		near "Homeward" 3 12
+		government "Coalition"
+	destination "Far Home"
+	on offer
+		conversation
+			`You're approached by a group consisting of a Saryd, Kimek, and Arach. The Saryd speaks and the small box around their neck translates after they stop.`
+			`	"The astonishing sojourner, with great joy we salute."`
+			`	You greet the group, and the Kimek continues the conversation, "The season of joy and camaraderie, we find ourselves in. Upon us, the Coalition games are."`
+			`	The Arach speaks last, asking, "To <destination>, we must go. Compete to our hardest, we will. Transport us, would you, if upon <payment>, you agreed on?"`
+			choice
+				`	"That sounds good. I'll bring you there."`
+					goto bbbb
+				`	"Sorry, I'm not looking for any transport jobs right now."`
+					decline
+				`	"Coalition games?"`
+			`	"To determine the athlete with the most merit, a competition it is," the Saryd says.`
+			`	"A candidate for the rank of Heliarch, one becomes, if one of the victors, one is," the Kimek adds.`
+			`	"Trained hard, we have. To prove our worth in the games, we wish," the Arach ends, and they once again look at you as if expecting an answer to their original question.`
+			choice
+				`	"Sounds like you're all really excited about this. Alright, I'll bring you there."`
+					goto bbbb
+				`	"Well I wish you luck then, but sadly I'm not looking for any transport jobs right now."`
+					decline
+			label bbbb
+			`	"Most grateful, we are," the Saryd says.`
+			`	As you help them bring their luggage into your ship, they introduce themselves: the Saryd is called Ulmodi, the Kimek is Pairri, and the Arach is Tumub. They settle in their bunks to rest for the upcoming event, though as soon as Pairri closes her bunk's door you hear some slight noises. It sounds like she's doing push-ups - or maybe squats; you can't really tell how a Kimek would name such exercises.`
+			`	Ulmodi explained that they need to get to <destination> (by <date>). It shouldn't be a problem considering you even have a Jump Drive to get there faster.`
+				accept
+	on complete
+		"coalition jobs" ++
+		payment 300000
+
+
+
+mission "Coalition Games 2"
+	invisible
+	landing
+	to offer
+		has "Spoder Bootle Centoro Athletes: done"
+	source "Far Home"
+	on offer
+		conversation
+			`As the athletes leave your ship and hand you your pay of 300,000 credits, Pairri stretches her legs, trying to get eye-level with you.`
+			`	You crouch, and she says, "Incapable of arriving on time, a mutual friend of ours was. Available, their ticket is."`
+			`	She extends one of her legs to you, holding a small, coin-like piece. It has a symbol like the old Olympic Games on Earth, except with three interlocked rings instead of five.`
```

Comment:
With everything in a single mission, you could use the text replacement here. I'd also condense these lines.
```suggestion
			`As the athletes leave your ship and hand you your pay of <payment>, Pairri stretches her legs, trying to get eye-level with you. You crouch, and she says, "Incapable of arriving on time, a mutual friend of ours was. Available, their ticket is." She extends one of her legs to you, holding a small, coin-like piece. It has a symbol like the old Olympic Games on Earth, except with three interlocked rings instead of five.`